node/cn/tracers/api.go

```diff
@@ -76,7 +76,7 @@ const (
 )
 
 var (
-	HeavyAPIRequestLimit int32 = 500
+	HeavyAPIRequestLimit int32 // HeavyDebugRequestLimitFlag
```

Comment:
Can you initialize this value as 500 and use it as default value?
```
        // 1. node/cn/tracers/api.go
        HeavyAPIRequestLimit int32 = 500
        // 2. cmd/utils/flags.go
	HeavyDebugRequestLimitFlag = &cli.IntFlag{
		Name:     "rpc.unsafe-debug.heavy-debug.request-limit",
		Usage:    "Limit the maximum number of heavy debug api requests. Works with unsafe-debug only.",
		Aliases:  []string{},
		EnvVars:  []string{"KLAYTN_RPC_UNSAFE_DEBUG_HEAVY_DEBUG_REQUEST_LIMIT"},
		Category: "API AND CONSOLE",
	}
        // 3. cmd/utils/config.go
	if ctx.IsSet(HeavyDebugRequestLimitFlag.Name) {
		tracers.HeavyAPIRequestLimit = int32(ctx.Int(HeavyDebugRequestLimitFlag.Name))
	}
```

Reply:
I thought of it too, but I think it can be confusing as well:
- users won't know the default value from "help" page
- updating `HeavyAPIRequestLimit` from source may be silently overridden

Do you have concern for this PR?

Reply:
No, I don't have concern. The test failed since `tracers.HeavyAPIRequestLimit` was not initialized so I want to fix it. 
If so, will you declare another default variable or use `tracers.HeavyAPIRequestLimit` as default value of the flag?

Reply:
I see. Then I'll just add back the initialization and a comment